cfgtool

```diff
@@ -285,14 +285,34 @@ Stay safe. Use crypto. Don´t drink and factorize."
 
 }
 
+#http://blog.pragbits.com/it/2015/02/09/slack-notifications-via-curl/
+__notify_slack() {    # format message as a code block ```${msg}```
+    SLACK_TITLE=$1
+    SLACK_BODY=$2
+    SLACK_URL="https://hooks.slack.com/services/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"
```

Comment:
HMM; I do not think this needs it's own variable, the key (or integration id)  should be red from a config file though.